lib/ulid/rails/formatter.rb

```diff
@@ -9,7 +9,7 @@ def self.format(v)
       end
 
       def self.unformat(v)
-        Base32::Crockford.decode(v).to_s(16).rjust(32, "0")
+        Base32::Crockford.decode(v.to_s).to_i.to_s(16).rjust(32, "0")
```

Comment:
I am on the fence about whether to allow non-string arguments to _try_ and find something matching in the database (as above, or e.g. `decoded_i = Integer(Base32::Crockford.decode(String(v)))` or if we should return a known non-existing ID instead? (`"0".rjust(32, "0")`?)

I guess the risk of matching a usable ID from `to_s`+`to_i` is minimal, right?